src/main/java/uk/gov/hmcts/reform/sscs/service/AirLookupService.java

```diff
@@ -56,25 +55,25 @@ public void init() {
         lookupAirVenueNameByPostCode = new HashMap<>();
         lookupVenueIdByAirVenueName = new HashMap<>();
 
-        String airlookupFilePath = "reference-data/AIRLookup9.xlsx";
+        String airlookupFilePath = "reference-data/AIRLookup10.xlsx";
         try {
             ClassPathResource classPathResource = new ClassPathResource(airlookupFilePath);
 
             OPCPackage pkg = OPCPackage.open(classPathResource.getInputStream());
-            Workbook wb2 = WorkbookFactory.create(pkg);
+            XSSFWorkbook wb2 = XSSFWorkbookFactory.create(pkg);
```

Comment:
WorkbookFactory is interface and which works for both xls and xlsx, if you use XSSWorkbookFactory then it will work for xlsx only. I think better to use WorkbookFactory

Reply:
yes, it makes sense. However, I changed that because the previous method is deprecated. You can see it yourself if you move onto the master branch. 

I tried to use the interface as you say but it did not work and it caused a lot of test failures. Then I read this: 
The Apache POI library supports both .xls and .xlsx files and is a more complex library than other Java libraries for working with Excel files.

It provides the Workbook interface for modeling an Excel file, and the Sheet, Row, and Cell interfaces that model the elements of an Excel file, as well as implementations of each interface for both file formats.

When working with the newer .xlsx file format, you would use the XSSFWorkbook, XSSFSheet, XSSFRow, and XSSFCell classes.

To work with the older .xls format, use the HSSFWorkbook, HSSFSheet, HSSFRow, and HSSFCell classes.
source: https://www.baeldung.com/java-microsoft-excel

And decided to leave it that way. In the end, we only use xlsx format here. 

I did change the variable though to use the interface which makes it more flexible in the line you suggested.